crates/apub/src/objects/mod.rs

```diff
@@ -46,6 +46,12 @@ pub(crate) fn read_from_string_or_source_opt(
     .map(|content| read_from_string_or_source(content, media_type, source))
 }
 
+pub(crate) fn append_attachments_to_comment(content: String, attachments: &[Attachment]) -> String {
+  attachments.iter().fold(content, |content, attachment| {
+    content + "  " + &attachment.as_markdown()
```

Comment:
Should probably use `\n\n` as separator.

Reply:
Two spaces at the end of a line gets turned into a linebreak, or `<br/>`. But this is missing the `\n` to do that.